app/views/clearance_mailer/change_password.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <p>
   <%= link_to t(".link_text", default: "Change my password"),
-    edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %>
+    edit_user_password_url(@user, token: @user.confirmation_token) %>
 </p>
 
-<p><%= raw t(".closing") %></p>
+<p><%= t(".closing") %></p>
```

Comment:
There doesn't appear to be anything in the translation which needs to be treated as a raw string: https://github.com/thoughtbot/clearance/blob/3d318088209b94c90ca29b31dc5310c635e7132c/config/locales/clearance.en.yml#L9-L10